config/config.go

```diff
@@ -10,7 +10,8 @@ import (
 
 // app meta
```

Comment:
warning: exported function PrintVersionAndExit should have comment or be unexported (golint)